tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Builders/OrganisationBuilder.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NHSD.BuyingCatalogue.Identity.Api.Models;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.UnitTests.Builders
+{
+    internal sealed class OrganisationBuilder
+    {
+        internal static OrganisationBuilder Create() => new OrganisationBuilder();
+
+        private Guid _organisationId;
+        private string _name;
+
+        public OrganisationBuilder()
```

Comment:
The static `Create` method mitigates the need for a public constructor. Could this be made private? This should simply the creation of this class as it will no longer have two methods to create an instance just one.